src/openapi.yaml

```diff
@@ -16,7 +16,26 @@ paths:
   /auth/callback:
     $ref: "./paths/auth/callback.yaml"
 
-  # TODO: add paths here
+  # /categories endpoints
+  /categories/{category_id}/entries:
+    $ref: "./paths/categories/category_id/entries.yaml"
+
+  /categories/{category_id}/entries/{entry_id}:
+    $ref: "./paths/categories/category_id/entry.yaml"
+
+  /categories/{category_id}/entries/create:
```

Comment:
We do not need an endpoint for every method. Instead we can use different methods on the same endpoint.

Use `POST` method on `/categories/{category_id}/entries`